README.md

```diff
@@ -1,3 +1,116 @@
-# whisper
+![alt text][logo]
+# Whisper subprotocol [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
 
 This repository gathers various whisper-related tools
+
+## Contents
+
+- [Whisper SubProtocol](#whisper-subprotocol)
+  - [Implementations](#implementations)
+  - [Specifications / White Papers](#specifications--white-papers)
+- [Applications](#applications)
+  - [Desktop Interfaces](#desktop-interfaces)
+  - [Web Interfaces](#web-interfaces)
+  - [Mobile applications](#mobile-applications)
+  - [Explorers](#explorers)
+  - [Misc](#misc)
+- [Developer Resources](#developer-resources)
+  - [Libraries](#libraries)
+  - [Tutorials](#tutorials)
+  - [Example Projects](#example-projects)
+- [Learning Resources](#learning-resources)
+  - [Talks](#talks)
+  - [Books](#books)
+- [Community](#community)
+  - [Telegram](#telegram)
```

Comment:
Let's not have that here. Telegram is a joke security-wise and it's a travesty that this "crypto" community uses it so much :wink: